README.md

```diff
@@ -14,14 +14,32 @@ including _Spy's Demise_.
 You'll need ROM files to use AppleToo (you can find ROMs pretty easily with
 a little Googling). 
 
+### Your own machine code
+
+Here's a minimal program, that just paints some dots in [hi-res
+graphics](https://www.xtof.info/blog/?p=768).
+Paste `a9558d3c22` into the Data field, `c000` into program counter, and
+then Run.
+
+### Disassembler
+
+You’ll need [CodeMirror](https://codemirror.net/) to use the disassembler.
+
+```
+git clone https://github.com/codemirror/CodeMirror.git
+cd CodeMirror
+npm build
+```
+
+
 ## TODO
 
  - Finish disk controller
  - Audio
  - Implement other graphics and text modes (low-res, mixed, etc)
  - 65C02 opcodes
 
-##Contributors
+## Contributors
 
   - Nicholas Bergson-Shilcock
```

Comment:
@raphlinus You should add yourself here :)

Reply:
Done